packages/shared/src/components/LoginButton.tsx

```diff
@@ -23,34 +21,37 @@ interface LoginButtonProps {
   icon?: React.ReactElement<IconProps>;
   className?: string;
 }
+
+enum ButtonCopy {
+  Login = 'Log in',
+  Signup = 'Sign up',
+}
+
 export default function LoginButton({
-  icon,
   className,
 }: LoginButtonProps): ReactElement {
   const { showLogin } = useContext(AuthContext);
-  const { flags } = useContext(FeaturesContext);
   const { trackEvent } = useContext(AnalyticsContext);
-  const buttonCopy = getFeatureValue(Features.SignupButtonCopy, flags);
-
-  useEffect(() => {
-    trackEvent(getAnalyticsEvent('impression', buttonCopy));
-    // @NOTE see https://dailydotdev.atlassian.net/l/cp/dK9h1zoM
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [buttonCopy]);
 
-  const onClick = () => {
-    trackEvent(getAnalyticsEvent('click', buttonCopy));
-    showLogin(AuthTriggers.MainButton);
+  const onClick = (copy: ButtonCopy) => {
+    trackEvent(getAnalyticsEvent('click', copy));
```

Comment:
Instead of tracking the copy here, let's change the target_type based on the button.
If it's a signup, let's keep it: `signup button`
For login let's use: `login button`